### PR TITLE
Update CITATION.cff file with 0.4.18 contributors

### PR DESCRIPTION
**Pull request by Czaki**
# Description

Update citation.cff to add people who contributed to napari 0.4.18 release.

### CODE REVIEW

**Comment by Nadalyn-CZI** [comment]
My information is: 

- given-names: Nadalyn
  family-names: Miller
  affiliation: Apex Systems
  orcid: https://orcid.org/0009-0007-6993-1267

**Review by jni — APPROVED** [review]

**Comment by kolibril13** [comment]
hi @Czaki , thanks for the notification on zulipchat, my information is:

given-names: Jan-Hendrik
family-names: Müller
affiliation: Georg-August-Universität Göttingen 
orcid: https://orcid.org/0009-0007-3670-9969

While going thorugh the list, I've noted that Contantin Pape is from the same university, but there the characters "ä" and "ö" are represented by their respective Unicode code:
affiliation: "Georg-August-Universit\xE4t G\xF6ttingen"

**Comment by Czaki** [comment]
> While going thorugh the list, I've noted that Contantin Pape is from the same university, but there the characters "ä" and "ö" are represented by their respective Unicode code:
> affiliation: "Georg-August-Universit\xE4t G\xF6ttingen"

I have fixed utf8 encoding problems

**Comment by Czaki** [comment]
@ClementCaporal @LucaMarconato @ctr26 @davidpross @dstansby @glichtner @gselzer @hmaarrfk @jeylau @jnahlers @jwindhager @kabilar @katherine-hutchings @kcpevey @ksofiyuk @lucyleeow @melissawm @p-j-smith

I'm updating the citation file by adding people who contribute to 0.4.18 and are not present in CITATION.cff file. 

Could You provide your data to add it here? This data will be used by a person who uses Cite this repository option and to create an entry on Zenodo.

**Comment by gselzer** [comment]
Thanks @Czaki!

given-names: Gabriel
family-names: Selzer
affiliation: University of Wisconsin-Madison
orcid: https://orcid.org/0009-0002-2400-1940

**Comment by ctr26** [comment]
Following above formal

given-names: Craig T.
family-names: Russell
affiliation: European Bioinformatics Institute - European Molecular Biology Laboratory
orcid: https://orcid.org/0000-0002-2447-5911

**Comment by jeylau** [comment]
Thank you @Czaki!

given-names: Jessy
family-names: Lauer
affiliation: Swiss Federal Institute of Technology (EPFL), Lausanne, Switzerland
orcid: https://orcid.org/0000-0002-3656-2449

**Comment by dstansby** [comment]
Thanks!

given-names: David
family-names: Stansby
affiliation: University College London
orcid: https://orcid.org/0000-0002-1365-1908

**Comment by p-j-smith** [comment]
Thank you!

given-names: Paul
family-names: Smith
affiliation: University College London
orcid: https://orcid.org/0000-0002-3676-5318

**Comment by davidpross** [comment]
Thanks @Czaki, here is my info:

given-names: David
family-names: Ross
affiliation: NanoString Technologies, Inc.
orcid: https://orcid.org/0000-0001-9998-3817

**Comment by katherine-hutchings** [comment]
Thanks! 😊
given-names: Katherine
family-names: Hutchings
affiliation: University College London
orchid: n/a

**Comment by kabilar** [comment]
Thanks!
given-names: Kabilar
family-names: Gunalan
affiliation: n/a
orchid: https://orcid.org/0000-0001-6964-4865

**Comment by jwindhager** [comment]
Thanks @Czaki 
```yaml
- given-names: Jonas
  family-names: Windhager
  affiliation: ETH Zurich / University of Zurich
  orcid: https://orcid.org/0000-0002-2111-5291
```

**Comment by lucyleeow** [comment]
Thanks @Czaki 

given-names: Lucy
family-names: Liu
affiliation: Quansight

**Comment by jnahlers** [comment]
Thanks!

given-names: Jannis
family-names: Ahlers
affiliation: Monash University
orcid: https://orcid.org/0000-0003-0630-1819

JANNIS AHLERS (he/him)
PhD Candidate

Monash University
School of Physics & Astronomy
Level G, New Horizons, Clayton campus
20 Research Way
Clayton, VIC 3800
Australia

E: ***@***.*** ***@***.***>
LinkedIn: linkedin.com/in/jannisahlers/
<https://www.linkedin.com/in/jannisahlers/>

monash.edu/science/schools/physics

We acknowledge and pay respects to the Elders and Traditional Owners of the
land on which our Australian campuses stand. Information for Indigenous
Australians <https://www.monash.edu/indigenous-australians>

We're committed to diversity and inclusion
<https://www.monash.edu/about/diversity-inclusion>

**Comment by glichtner** [comment]
Thanks @Czaki!

given-names: Gregor
family-names: Lichtner
affiliation: Universitätsmedizin Greifswald
orcid: https://orcid.org/0000-0002-5890-1958

**Comment by ClementCaporal** [comment]
Thanks @Czaki

```yaml
- given-names: Clément
  family-names: Caporal
  affiliation: Laboratory for Optics and Biosciences, Ecole Polytechnique, INSERM, CNRS, Palaiseau, France
  orcid: https://orcid.org/0000-0002-9441-9173
```

**Comment by hmaarrfk** [comment]
```
- given-names: Mark
  family-names: Harfouche
  affiliation: Ramona Optics Inc, Durham, North Carolina, USA
  orcid: https://orcid.org/0000-0002-4657-4603
```

**Comment by Czaki** [comment]
@kcpevey @ksofiyuk I have added you base on github information


@LucaMarconato There is no enough information on github to add you to citation cff file

**Comment by Czaki** [comment]
Should I try to add personal alias to an easier mapping person on GitHub nick?  https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#definitionspersonalias

**Comment by Carreau** [comment]
> Should I try to add personal alias to an easier mapping person on GitHub nick? https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#definitionspersonalias

I think it will only make it easier to find who need to be added later on so that's a yes for me.

**Comment by LucaMarconato** [comment]
@thanks Czaki, here is my info:

```
- given-names: Luca
  family-names: Marconato
  affiliation: EMBL Heidelberg
  orcid: https://orcid.org/0000-0003-3198-1326
```

**Comment by Czaki** [comment]
@orena1 @Chris-N-K 
Both of You contribute to docs that are backported to 0.4.18. COuld you check your data?

**Comment by Chris-N-K** [comment]
> @orena1 @Chris-N-K Both of You contribute to docs that are backported to 0.4.18. Could you check your data?

All fine thanks for your work! :)

File: CITATION.cff [hunk 1]
```diff
@@ -1,111 +1,271 @@
...
+  affiliation: Monash University
+  orcid: orcid.org/0000-0003-0630-1819
+  alias: jnahlers
+- given-names: Daniel
+  family-names: Althviz Moré
+  affiliation: Quansight
+  orcid: https://orcid.org/0000-0003-1759-4194
+  alias: dalthviz
+- given-names: Oren
+  family-names: Amsalem
```
Comment (orena1) [inline CITATION.cff:17]: affiliation: Harvard Medical School, BIDMC
orcid:  https://orcid.org/0000-0002-8070-0378

**Review by orena1 — CHANGES_REQUESTED** [review]

**Comment by orena1** [comment]
Thanks a lot @Czaki , I've added my affiliation and orcid. 

Harvard Medical School, BIDMC
https://orcid.org/0000-0002-8070-0378

File: CITATION.cff (hunk 1 above)
Comment (Czaki) [inline CITATION.cff:17]:
```suggestion
- given-names: Oren
  family-names: Amsalem
  affiliation: Harvard Medical School, BIDMC
  orcid: orcid.org/0000-0002-8070-0378
```

**Comment by goanpeca** [comment]
@Czaki my details :)

```
- given-names: Gonzalo
  family-names: Peña-Castellanos
  affiliation: Quansight
  orcid: https://orcid.org/my-orcid?orcid=0000-0002-1214-4680
```

**Pull request merged**